File: Data/Orchester.cs
using System.ComponentModel.DataAnnotations;

namespace MaestroNotes.Data
{
    public class Orchester
    {
        [Key]
        public int Id { get; set; }
        [MaxLength(100)]
        public string Name { get; set; } = "";
        public DateTime? Founded { get; set; }
        [MaxLength(1000)]
        public string? Note { get; set; }

    }
}

File: php-app/app/Models/Orchester.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Orchester extends Model
{
    use HasFactory;

    protected $table = 'orchester';

    protected $fillable = [
        'name',
        'founded',
        'note',
    ];

    protected $casts = [
        'founded' => 'date',
    ];
}
